packages/utils/src/eslint-plugin/selectors.ts

```diff
@@ -60,14 +62,26 @@ export const INPUTS_METADATA_PROPERTY_LITERAL = `${COMPONENT_OR_DIRECTIVE_CLASS_
   'inputs',
 )} > ArrayExpression ${LITERAL_OR_TEMPLATE_ELEMENT}`;
 
-export const INPUT_ALIAS = `:matches(PropertyDefinition, MethodDefinition[kind='set']) ${INPUT_DECORATOR} ${LITERAL_OR_TEMPLATE_ELEMENT}`;
+export const INPUT_ALIAS = [
+  `:matches(PropertyDefinition, MethodDefinition[kind='set']) ${INPUT_DECORATOR} > CallExpression > Literal`,
+  `:matches(PropertyDefinition, MethodDefinition[kind='set']) ${INPUT_DECORATOR} > CallExpression > TemplateLiteral > TemplateElement`,
```

Comment:
ℹ️ I didn't use `LITERAL_OR_TEMPLATE_ELEMENT` here. Using that selector means the `no-input-rename` rule would need to check if the string that matched is the value of an `alias` property to avoid false positives. By using three different selectors, it guarantees that it matches to either `@Output('foo')` or ``@Output(`foo`)`` or `@Output({ alias: 'foo' })`, and _won't_ match to something like `@Output({ test: 'foo' })`.

You'll see in the `no-input-rename` rule that I've been able to remove the two checks where it was looking for the parent property or object expression to determine if the matched literal was the value of an `alias` property.